template/prism-theme-template.css

```
@@ -0,0 +1,209 @@
+/**
+ * Your theme's name
+ * If this is an adaptation of an existing theme, credit the creator and/or leave a link to it!
+ * Optional: Your name and/or username
+ */
+
+/**
+ * Prism supports IE11, which does not support CSS variables
+ * However, you are encouraged to leave a list of colours you use here
+ * so that when we transition to Prism V2 (and drop support for IE11),
+ * the transition will be a little easier!
+ */
+
```

I think we should probably tell people how to use this template.

```suggestion

/**
 * How to use this template:
 *
 * This file contains all the boilerplate necessary for a Prism theme along with template rules for you to fill in.
 *
 * All properties with the value `unset` are for you to change.
 * You should fill in all `color` and `background` properties.
 * If you don't need an `unset` property (e.g. `border-radius`), then feel free to remove it.
 *
 * Your finished theme should have all `unset` properties either filled in or removed.
 */

```